third_party/nvfuser/csrc/executor.cpp

```diff
@@ -1389,7 +1426,8 @@ void FusionExecutor::compileRtc(
   FUSER_PERF_SCOPE("ExecutorRunFusion::compileRtc");
   std::string scode;
   if (!structured) {
-    scode = getStructuredCode(code);
+    // Assumes the code string uses int64 indexing
```

Comment:
`compileRtc` is only used for debugging? It would be nice to `TORCH_WARN` about this behavior because we might be using handwritten kernel doing perf tuning.

Reply:
Yes. It's also used by some of the C++ tests. I don't like to see warnings every time the tests are run, so I'll change the interface to require the index type always.